contributing/contributing-code.md

```diff
@@ -179,6 +179,8 @@ These guidelines are specific to the amphtml repository. Other ampproject repos
   - Owners of an area may approve other Owners at or below their area of expertise following the normal PR process. To request becoming an Owner create a PR adding yourself to the appropriate OWNERS file and assign/cc a current Owner for that directory.
 - The list of Owners for a directory can be found in the [OWNERS](https://github.com/ampproject/amphtml/search?o=asc&q=filename%3A"OWNERS"&s=indexed) file in the directory or a parent directory.
 
+Please note that some Owners may be relevant regardless of code location and therefore may include one or more [AMP Working Groups](https://amp.dev/community/governance/#working-groups), such as the groups for [Security & Privacy](https://amp.dev/community/working-groups/security-privacy/) and [User Interface & Accessibility](https://amp.dev/community/working-groups/ui-and-a11y/). 
```

Comment:
Looks like prettier is complaining about the space at the end of the line -- apologies for the delay. 

Once the checks run again I'd be happy to merge. :) 

@rcebulko what is the deal with CLA? Can we just update the label or is there more to it? 
```suggestion
Please note that some Owners may be relevant regardless of code location and therefore may include one or more [AMP Working Groups](https://amp.dev/community/governance/#working-groups), such as the groups for [Security & Privacy](https://amp.dev/community/working-groups/security-privacy/) and [User Interface & Accessibility](https://amp.dev/community/working-groups/ui-and-a11y/).
```

Reply:
This is the new OpenJS CLA, the old Google CLA bot is deprecated now that AMP is part of JSF

Reply:
See https://github.com/ampproject/meta/issues/60#issuecomment-705851924

Reply:
I believe the new OpenJS CLA is still non-blocking. It looksl ike the Google CLA is still what is needed to submit this PR. :)

Reply:
@patrickhlauke @TetraLogicalHelpdesk The PR also needs the suggested change above (remove white space) :) to pass our CI, thanks!